### ISSUE TITLE
unable to pull data using my league ID

### ISSUE THREAD

**Issue by cdenny** [isaacmiller2004/GetYahooFBBData#2]
found some interesting behavior,  think it may be yahoo side blocking these kinds of scripts

When trying to get data using my league Id I get a yahoo login returned (wrote to a file what BS is getting  returned)

however if I use a league id of 1 I can pull data back, even though league 1 returns no data in a browser.

**Comment by isaacmiller2004** [comment]
Yeah, I think it will only work if you set the "Allow apps that use less secure sign in" flag in Yahoo account security settings.  Up to you as to whether it's worth it.
